charts/seqr/values.yaml

```diff
@@ -169,7 +171,13 @@ checkForNewSamplesJobEnabled: false
 
 redis:
   # -- enables or disables redis deployment using this chart
-  enabled: false
+  enabled: true
+
+  # -- default redis settings
+  architecture: standalone
+  auth:
+    enabled: no
+  fullnameOverride: "seqr-redis"
```

Comment:
We're getting `seqr-redis` in our deployment right now because we're using `seqr` as our `.Release.Name`.  This will change for both us and local users (`broad` for us, `institution name` for local users).